sw/device/lib/base/log_dv.h

```diff
@@ -0,0 +1,35 @@
+// Copyright lowRISC contributors.
```

Comment:
None of this is specific to DV. If we want to keep these macros around (not sure we do, honestly...), they should be kept in a more... generic header, like "macros.h".